rust/rsc/src/bin/rsc/types.rs

```diff
@@ -172,3 +173,27 @@ pub enum ReadJobResponse {
 pub struct GetUploadUrlResponse {
     pub url: String,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DashboardStatsOldestJob {
+    pub label: String,
+    pub created_at: DateTime,
+    pub savings: i64,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DashboardStatsMostReusedJob {
+    pub label: String,
+    pub reuses: i32,
+    pub savings: i64,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DashboardStatsResponse {
+    pub job_count: u64,
+    pub blob_count: u64,
+    pub size: i64,
+    pub savings: i64,
```

Comment:
Can savings and size be negative? Why wouldn't they also be `u64`?
EDIT: oh I think this is just the SQL doesn't have a u64 type again.

Reply:
yep! I'm not actually sure why job_count/blob_count are allowed to be u64 here, I think its because summing over an i64 can overflow the container